src/components/PagesList/index.jsx

```diff
@@ -1,7 +1,38 @@
 import React from "react";
+import { Pagination } from "react-bootstrap";
+import allNgos from "../NgoList/allNgos";
 
-const PagesList = () => {
-  return <div></div>;
+const PagesList = ({ ngos = allNgos }) => {
+  const ngosPerPage = 10;
+  const pageNumbers = [];
+  for (let i = 1; i <= Math.ceil(ngos.length / ngosPerPage); i++) {
+    pageNumbers.push(i);
+  }
+
+  return (
+    <div className="pages">
+      <Pagination
+        className="d-flex justify-content-end pb-5"
+        style={{
```

Comment:
lets use classes instead of inline styles to keep the consistency of the code

Reply:
Actually I tried to do that but I couldn't select the background of the pagination bootstrap component even using the dev tools of chrome.